### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by Lokeshkhati**
image

Screen for cart managemant

### CODE REVIEW

File: components/pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.cart-container {
+    flex-wrap: wrap-reverse;
+    align-items: flex-end;
```
Comment (nakulsharma15) [inline components/pages/cart/cart.css:3]: You can extract this in a utility class so that it can be used elsewhere as well

**Review by nakulsharma15 — APPROVED** [review]
Gave a small suggestion. Everything else seems good.

File: components/pages/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  
+  @media screen and (min-width: 648px) {
+    .cart-card {
+      margin: 0 0 5rem 0;
+      flex-wrap: nowrap;
+    }
+  }
+  
+  
+
```
Comment (tanishq20) [inline components/pages/cart/cart.css:43]: Avoid giving extra spaces.

**Review by tanishq20 — APPROVED** [review]
Clean code

**Pull request merged**